packages/next-swc/crates/next-core/src/next_edge/context.rs

```diff
@@ -63,15 +65,10 @@ async fn next_edge_free_vars(
     Ok(free_var_references!(
         ..defines(&*define_env.await?).into_iter(),
         Buffer = FreeVarReference::EcmaScriptModule {
-            request: "next/dist/compiled/buffer".to_string(),
+            request: "buffer".to_string(),
             lookup_path: Some(project_path),
             export: Some("Buffer".to_string()),
         },
-        process = FreeVarReference::EcmaScriptModule {
```

Comment:
https://github.com/vercel/next.js/blob/160bb99b06e9c049f88e25806fd995f07f4cc7e1/packages/next/src/build/webpack-config.ts#L1718

in webpack this is client only, does not apply to edge layer.